tests/test_todoitems_routes.py

```diff
@@ -0,0 +1,92 @@
+from tests.test_main import client
+
+
+def test_get_all_empty_list():
+    response = client.get("/todoitems/")
+    assert response.status_code == 200
+    assert response.json() == []
+
+
+def test_get_all_todoitems():
+    new_todoitem1 = {"name": "fastapi", "description": "Fastapi education"}
+    client.post(
+        "/todoitems/create/",
```

Comment:
So if this app would have been using a real database, you would insert data into it everytime you run a test? :) 
That's not good.   Use monkeypatch to swap  your todoitem_list from the todoitem_service.py with something that using mock data.